.github/workflows/deploy.yml

```
@@ -0,0 +1,93 @@
+      - name: Verify Deployment
+        run: |
+          ssh -i ~/.ssh/deploy_key ${{ secrets.DEPLOY_USER }}@${{ secrets.DEPLOY_HOST }} "
+            cd ~/deploy && make status
+          "
+
+      - name: Cleanup old images (if successful)
+        if: success()
+        run: |
+          ssh -i ~/.ssh/deploy_key ${{ secrets.DEPLOY_USER }}@${{ secrets.DEPLOY_HOST }} "
+            docker system prune -af --filter 'until=168h'
+          " 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)